opwen_email_server/celery/celery.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+from celery import Celery
+
+celery = Celery(include = [ "opwen_email_server.api.store_written_client_emails",
+                            "opwen_email_server.api.send_outbound_emails",
+                            "opwen_email_server.api.store_inbound_emails"])
+
+class Config:
+    BROKER_URL = "amqp://"
```

Comment:
Please make these configurable via environment variables so that we can swap them out easily for tests/production use-cases. E.g. for testing we likely will want a local rabbitmq server but in production we'll likely want to use ServiceBus since it's managed. You can add the environment variables to [config.py](https://github.com/ascoderu/opwen-cloudserver/blob/5be793ee10c7a59443625c74ab83fe650d91f8c3/opwen_email_server/config.py) where we gather all of the other configuration values to make it very easy to see in one place all the knobs that can be turned to affect the system.

Reply:
Will do, thanks for the insight!